src/pages/Login/Login.tsx

```diff
@@ -59,6 +65,20 @@ const Login: React.FC = () => {
               helpText={formik.touched.password && formik.errors.password}
             />
             {error && <div className="error text-red-400">{error}</div>}
+
```

Comment:
This checkbox need to come above the line 61.

Reply:
done